runners-provisioning/test-runner.yml

```diff
@@ -0,0 +1,65 @@
+- hosts: RUNNERVM
+  tasks:
+  - name: Change hostname to centos-68-test
+    hostname:
+      name: "centos-68-test"
+    become: true
+
+  - name: Modify /etc/hosts
+    lineinfile:
+      dest: /etc/hosts
+      regexp: '^127\.0\.0\.1[ \t]+localhost'
+      line: '127.0.0.1 localhost centos-68-test'
+      state: present
+    become: true
+
+  - name: Remove repository (and clean up left-over metadata)
+    yum_repository:
+      name: wandisco
+      state: absent
+    become: true
```

Comment:
Why we remove this? If it is present and it is not the version we want we should update the other script.